### DISCUSSION TITLE
Installation without full CLI access possible?

### DISCUSSION THREAD

**Discussion by Zodiac1978** [Kovah/LinkAce#416] [General]
I would like to test LinkAce. It looks great and I very much like the philosophy. But my shared hosting does not have full CLI access. Is there any chance to install LinkAce without using the command line?

**Comment by Kovah** [comment]
Well, you can try it. The App Key could possibly be generated online (https://keygen.io/).
But there will be no support for any issues occurring with this method. Also, no CLI commands will work which are helpful in some situations.
